_data/portals/admin.json

```diff
@@ -142,7 +142,8 @@
       },
       {
         "portalName": "Power Automate Maker Portal",
-        "primaryURL": "https://flow.microsoft.com"
+        "primaryURL": "https://flow.microsoft.com",
```

Comment:
Also a secondary URL exists, but is not noted here: http://powerautomate.microsoft.com/